x/ethbridge/keeper/peggyTokens.go

```diff
@@ -15,14 +15,14 @@ func (k Keeper) AddPeggyToken(ctx sdk.Context, token string) {
 	store := ctx.KVStore(k.storeKey)
 	key := types.PeggyTokenKeyPrefix
 
-	tokens = append(tokens, token)
-	store.Set(key, k.cdc.MustMarshalBinaryBare(tokens))
+	tokens.Tokens = append(tokens.Tokens, token)
```

Comment:
Why do we need to make this change to the way we store data in the keeper? We should be able to do it the old way and just append to an array of type tokens without having to use a property of the object to append to.